app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,44 @@
+class LineItemsController < ApplicationController
+  before_action :authorize!
+  def new
+    if user_last_order.paid == true
+      @order = current_user.order.create
+      @line_item = @order.new
+    else
+      @line_item = user_last_order.line_item.new
```

Comment:
wouldn't this indicate that if there is an open order, the current line item is the column of line items on the last order in the database?